plugins/csv-parser/src/headerDetect.ts

```diff
@@ -22,20 +22,20 @@ const getType = (value: string | number): string => {
 			.replace(/[\u2012\u2013\u2014\u2015]/g, '-')
 
 		return !isNaN(Number(formattedValue)) ? 'number' : 'string'
```

Comment:
Maybe the whole getType fn is duplicated?
https://github.com/vizzuhq/vizzu-lib-ext/pull/58/files#diff-630ed1549ccd46bc82d93e2a9323013ba377fa3fabc45e473d616b9747615779R18

Reply:
because they are two separate plugins completely independent from each other.